Index: src\core\scene_stack_ex.c
===================================================================
--- src\core\scene_stack_ex.c
+++ src\core\scene_stack_ex.c
@@ -15,9 +15,8 @@
 #include "data_manager.h"
 #include "vm_exceptions.h"
 #include "input.h"
 #include "meta_tiles.h"
-#include "scene_transition.h"
 
 typedef struct vm_stack_item_t {    
     // VM contexts
     SCRIPT_CTX CTXS[VM_MAX_CONTEXTS];
@@ -86,17 +85,8 @@
     unsigned char* metatile_collision_ptr;
     UBYTE image_tile_width_bit;
 } meta_tiles_stack_item_t;
 
-typedef struct scene_transition_stack_item_t {
-    //scene transition
-    UBYTE scene_transition_enabled;
-    far_ptr_t up_scene;
-    far_ptr_t right_scene;
-    far_ptr_t down_scene;
-    far_ptr_t left_scene;    
-} scene_transition_stack_item_t;
-
 typedef struct scene_stack_item_ex_t {
     far_ptr_t scene;
     upoint16_t player_pos;
     direction_e player_dir;
@@ -106,9 +96,8 @@
     actor_stack_item_t actor_stack_item;
     projectile_stack_item_t projectile_stack_item;
     camera_stack_item_t camera_stack_item;
     meta_tiles_stack_item_t meta_tiles_stack_item;
-    scene_transition_stack_item_t scene_transition_stack_item;
     uint16_t rand_seed;
 } scene_stack_item_ex_t;
 
 extern uint16_t __rand_seed;
@@ -193,17 +182,8 @@
     meta_tiles_stack_item->metatile_collision_ptr = metatile_collision_ptr; 
     meta_tiles_stack_item->image_tile_width_bit = image_tile_width_bit;
 }
 
-static void push_scene_transition_stack_item(void){
-    scene_transition_stack_item_t* scene_transition_stack_item = &scene_stack_ex_ptr->scene_transition_stack_item;
-    scene_transition_stack_item->scene_transition_enabled = scene_transition_enabled;    
-    scene_transition_stack_item->up_scene = up_scene;
-    scene_transition_stack_item->right_scene = right_scene;
-    scene_transition_stack_item->down_scene = down_scene;
-    scene_transition_stack_item->left_scene = left_scene;    
-}
-
 void push_scene_stack_ex(void) BANKED {
     if (scene_stack_ex_count < MAX_SCENE_STACK_EX_COUNT){
         SWITCH_RAM_BANK(1, RAM_BANKS_ONLY);
         scene_stack_ex_ptr->scene = current_scene;
@@ -215,9 +195,8 @@
         push_actor_stack_item();
         push_projectile_stack_item();
         push_camera_stack_item();
         push_meta_tiles_stack_item();
-        push_scene_transition_stack_item();
         scene_stack_ex_ptr->rand_seed = __rand_seed;
         scene_stack_ex_ptr++;
         scene_stack_ex_count++;
         SWITCH_RAM_BANK(0, RAM_BANKS_ONLY);
@@ -305,17 +284,8 @@
     metatile_collision_ptr = meta_tiles_stack_item->metatile_collision_ptr;
     image_tile_width_bit = meta_tiles_stack_item->image_tile_width_bit;
 }
 
-static void pop_scene_transition_stack_item(void){
-    scene_transition_stack_item_t* scene_transition_stack_item = &scene_stack_ex_ptr->scene_transition_stack_item;
-    scene_transition_enabled = scene_transition_stack_item->scene_transition_enabled;    
-    up_scene = scene_transition_stack_item->up_scene;
-    right_scene = scene_transition_stack_item->right_scene;
-    down_scene = scene_transition_stack_item->down_scene;
-    left_scene = scene_transition_stack_item->left_scene;    
-}
-
 UBYTE pop_scene_stack_ex(void) BANKED {
     if (scene_stack_ex_count > 0){
         SWITCH_RAM_BANK(1, RAM_BANKS_ONLY);
         scene_stack_ex_count--;
@@ -330,9 +300,8 @@
         pop_actor_stack_item();
         pop_projectile_stack_item();
         pop_camera_stack_item();
         pop_meta_tiles_stack_item();
-        pop_scene_transition_stack_item();
         __rand_seed = scene_stack_ex_ptr->rand_seed;
         
         SWITCH_RAM_BANK(0, RAM_BANKS_ONLY);
         return TRUE;
