Index: src\core\data_manager.c
===================================================================
--- src\core\data_manager.c
+++ src\core\data_manager.c
@@ -60,13 +60,14 @@
     scene_stack_ptr = scene_stack;
 }
 
 void load_bkg_tileset(const tileset_t* tiles, UBYTE bank) BANKED {
-	#ifdef DISABLE_TILESET_LOAD_ON_TRANSITION
+    #ifdef DISABLE_TILESET_LOAD_ON_TRANSITION
         if (((!bank) && (!tiles)) || is_transitioning_scene) return;    
-        #else
+    #else
         if ((!bank) && (!tiles)) return;
     #endif
+
     UWORD n_tiles = ReadBankedUWORD(&(tiles->n_tiles), bank);
 
     // load first background chunk, align to zero tile
     UBYTE * data = tiles->tiles;
@@ -345,8 +346,9 @@
     // Load triggers
     if (triggers_len != 0) {
         MemcpyBanked(&triggers, scn.triggers.ptr, sizeof(trigger_t) * triggers_len, scn.triggers.bank);
     }
+
     scene_transition_reset();
     scroll_reset();
     scroll_offset_x = 0;
     scroll_offset_y = 0;
